sdk/helper/ldaputil/config.go

```diff
@@ -464,7 +464,7 @@ type ConfigEntry struct {
 	RequestTimeout           int    `json:"request_timeout"`
 	ConnectionTimeout        int    `json:"connection_timeout"`
 	DerefAliases             string `json:"dereference_aliases"`
-	MaximumPageSize          int    `json:"max_page_size"`
+	MaximumPageSize          *int   `json:"max_page_size"`
```

Comment:
I propose this should stay as an `int`, as the zero value is not valid configuration, so there is no need to discriminate between "set to 0" and "not set".

Reply:
Agreed.